### DISCUSSION TITLE
spelling: scheduler argument max_intances

### DISCUSSION THREAD

**Discussion by devkral** [dymmond/asyncz#49] [Potential Issue]
The parameter max_intances should maybe be renamed to max_instances.
The old name can be deprecated.

Otherwise it will become a source of confusion

**Comment by tarsil** [comment]
You are absolutely correct and I'm even shocked that I let that one slide. 

During the next week I will update that for sure.

Thank you once again @devkral

**Comment by tarsil** [comment]
> The parameter max_intances should maybe be renamed to max_instances. The old name can be deprecated.
> 
> Otherwise it will become a source of confusion

Thank you for this. This was now addressed in the version 0.7.0.
